### PR TITLE
feat: exclude F1 doctors from reval sync

### PR DESCRIPTION
**Pull request by CaiWillis**
TIS21-4459

### CODE REVIEW

**Review by hihilary — CHANGES_REQUESTED** [review]
Got some questions to think about:
1. it's `Placement.traineeId` to check with `ContactDetails.id`
2. how do we handle null values (a person doesn't have any placements)
3. if a person has multiple placement, how we guarantee we export only 1 line

**Comment by CaiWillis** [comment]
Will have a think and come back with some better ideas

File: tcs-persistence/src/main/resources/queries/traineeInfoForConnection.sql [hunk 1]
```diff
@@ -17,6 +17,8 @@ from (
     -- note: null values are filtered out by the condition below
     and lower(gmc.gmcNumber) <> 'unknown'
     and gmc.gmcNumber not like CONCAT('%', UNHEX('c2a0'), '%') -- filter out all gmc number with non-breaking space
+  inner join Placement pl on (pl.id = cd.id )
```
Comment (JosephKelly) [inline tcs-persistence/src/main/resources/queries/traineeInfoForConnection.sql:20]: Requires change: `Placement.id` should be replaced with `Placement.traineeId` and the scope limited to "current" placements.

If not already considered: please describe if doctors without a current placement are intentionally excluded.

Comment (JosephKelly) [reply tcs-persistence/src/main/resources/queries/traineeInfoForConnection.sql:20]: Too slow :turtle:

**Pull request closed without merging**